islandora_videojs.module

```diff
@@ -64,9 +66,12 @@ function template_preprocess_islandora_videojs(array &$variables) {
   if (isset($params['sources']) && is_array($params['sources'])) {
     $variables['sources'] = $params['sources'];
   }
-  else {
+  elseif (isset($params['url']) && isset($params['mime'])) {
     $variables['sources'][] = array('url' => $params['url'], 'mime' => $params['mime']);
   }
+  else {
+    $variables['sources'] = array();
```

Comment:
I would avoid the `else` at line 72  at all and just initialize `$variables['sources'] = array();` at the beginning of the function.  We could also, simply drop adding the libraries if sources is empty at all, both in the preprocess and the tpl. file? and just display the poster frame if set? Plus the message @whikloj correctly suggests which means i also support adding that message. Only, minor note, i would discourage any messages that encourage contacting the site administrator, which is for sure someone unknown for an anonymous visitor.